### PR TITLE
Include example for exclude syntax

### PR DESCRIPTION
**Pull request by robdodson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ for this step).
 - `-h`|`--help`: print this message
 - `-v`|`--version`: print version number
 - `-p <arg>`|`--abspath <arg>`: use <arg> as the "webserver root", make all adjusted urls absolute
-- `--exclude <path>`: exclude a subpath from root. Use multiple times to exclude multiple paths. Tags to excluded paths are kept.
+- `--exclude <path>`: exclude a subpath from root. Use multiple times to exclude multiple paths. Tags to excluded paths are kept. ex: `--exclude=elements/x-foo.html`
```
Comment (ebidel) [inline README.md]: How about a multiple exampel: `--exclude=elements/x-foo.html --exclude=elements/x-foo2.html`

Comment (ebidel) [inline README.md]: Can we update to a better description. 

"Tags (imports/scripts/etc) that reference an excluded path are left in-place, meaning the resources are not inlined"

**Comment by robdodson** [comment]
Updated. @ebidel PTAL

**Comment by azakus** [comment]
LGTM

**Pull request merged**